### PR TITLE
Implement includes() for eslevel: 2016

### PR DESCRIPTION
**Pull request by onnimonni**
This implements #44.

### CODE REVIEW

File: spec/functions_spec.rb [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 describe Ruby2JS::Filter::Functions do
-  
-  def to_js( string)
+
+  def to_js(string)
     Ruby2JS.convert(string, filters: [Ruby2JS::Filter::Functions]).to_s
   end
-  
+
+  def to_es2016(string)
```
Comment (rubys) [inline spec/functions_spec.rb]: At the moment, the eslevel specific tests are in spec/es201*_spec.rb.  Can this be moved to es2016_spec?  You can steal/adapt the to_js_fn(string) function from es2017_spec.

File: lib/ruby2js/filter/functions.rb [hunk 2]
```diff
@@ -203,8 +203,12 @@ def on_send(node)
             S(:and, s(:send, args.first, :>=, target.children.first),
               s(:send, args.first, :<, target.children.last))
           else
-            process S(:send, S(:send, target, :indexOf, args.first), :!=,
-              s(:int, -1))
+            if es2016
+              process S(:send, target, :includes, args.first)
```
Comment (rubys) [inline lib/ruby2js/filter/functions.rb]: Can you mention this in the README?  Ideally in this following section: https://github.com/rubys/ruby2js/blob/dd635d5af02e72d65564f496171510cbae47aefd/README.md#es2016-support

File: spec/functions_spec.rb (hunk 1 above)
Comment (onnimonni) [reply spec/functions_spec.rb]: Of course!

Comment (onnimonni) [reply spec/functions_spec.rb]: Tell me is there something that should be changed, I wasn't sure if you want to have a second function or add additional filters into those tests.

File: lib/ruby2js/filter/functions.rb (hunk 2 above)
Comment (onnimonni) [reply lib/ruby2js/filter/functions.rb]: Done

File: spec/functions_spec.rb (hunk 1 above)
Comment (onnimonni) [reply spec/functions_spec.rb]: Sorry I did not listen to this part:

> You can steal/adapt the to_js_fn(string) function from es2017_spec.

**Pull request merged**